group/assignment6/move_cars.py

```diff
@@ -68,7 +80,8 @@ def generate_moves(start: List[int], final: List[int]) -> List[int]:
             yield car, place, empty_place
             empty_place = _move(car_positions, car)
 
-        # move car that should end up in current empty space while such car exists
+        # move car that should end up in current empty space while such a car
+        # exists
```

Comment:
This break looks a bit weird with one word on a new line

Reply:
Yep, I just don't know how to break it better if we should keep each line <= 80 characters